backend/src/main/java/ca/bc/gov/hlth/hnweb/security/SecurityConfig.java

```diff
@@ -39,6 +39,7 @@ protected void configure(HttpSecurity http) throws Exception {
             .mvcMatchers(HttpMethod.GET, "/docs/**").permitAll()
             .mvcMatchers(HttpMethod.POST,"/eligibility/check-msp-coverage-status").hasRole("E45")
             .mvcMatchers(HttpMethod.POST,"/eligibility/check-eligibility").hasRole("R15")
+            .mvcMatchers(HttpMethod.POST,"/eligibility/inquire-phn").hasRole("R41")
```

Comment:
Should we have constants or enum for all the v2 types?

Reply:
Possibly. But we may end up renaming them in the future. Also, with all the String matching happening in this class I didn't feel that moving everything to constants was worth the loss of simplicity.

Reply:
Yeah, it looks clean the way it is.